client/components/PrivateRoute.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+
+export const PrivateRoute = ({ component: Component, ...rest }) => (
+    <Route {...rest} render={props => (
+        localStorage.getItem('user')
```

Comment:
so is this just relying on _something_ being populated in localStorage at `user` in order to "privatize" the route? 

it doesn't seem secure to me but also i don't think it's worth changing for a hackathon

Reply:
Hi @addelong, thanks for you support! All typos are fixed now. About the "privatize", I agree it's not the best approach at all, but I once it's on the client side and all real validation occurs on the server side, I did see any problems on that for the Hackathon. Once it's became a real project, it will not be a big deal changing that.